independent-publisher-connectors/My Acclaro/apiDefinition.swagger.json

```diff
@@ -0,0 +1,13061 @@
+                    },
+                    "duedate": {
+                      "type": "string",
+                      "description": "duedate"
+                    },
+                    "clientref": {
+                      "type": "string",
+                      "description": "clientref"
+                    },
+                    "delivery": {
+                      "type": "string",
+                      "description": "delivery"
+                    },
+                    "rating": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "rating"
+                    },
+                    "review": {
+                      "type": "string",
+                      "description": "review"
+                    },
+                    "created": {
+                      "type": "string",
+                      "description": "created"
+                    },
+                    "modified": {
+                      "type": "string",
+                      "description": "modified"
+                    },
+                    "sourcelang": {
+                      "type": "array",
+                      "items": {},
+                      "description": "sourcelang"
+                    },
+                    "targetlang": {
+                      "type": "array",
+                      "items": {},
+                      "description": "targetlang"
+                    },
+                    "ordertypecode": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "ordertypecode"
+                    },
+                    "emailaddress": {
+                      "type": "string",
+                      "description": "emailaddress"
+                    },
+                    "emailquote": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "emailquote"
+                    },
+                    "emailfile": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "emailfile"
+                    },
+                    "emaildone": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "emaildone"
+                    },
+                    "emailsubmit": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "emailsubmit"
+                    },
+                    "emailcomment": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "emailcomment"
+                    },
+                    "quoterequired": {
+                      "type": "integer",
+                      "format": "int32",
+
```

Comment:
@msanz-acclaro Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

Reply:
corrected